server/index.js

```diff
@@ -0,0 +1,28 @@
+const express = require('express');
+const db = require('../database/index');
+const bodyParser = require('body-parser');
+require('../aDataGenerator/dataGenerator');
+
+const app = express();
+
+app.use(express.static(__dirname.concat('../public')));
+app.use(bodyParser.json());
+
+app.post('/homes', (req, res) => {
+  db.save(req.body);
+  res.sendStatus(201);
```

Comment:
I think the sendStatus should technically be a callback to db.save function so that it only sends the 201 if the save to the database is successful.